modules/node-shared/src/main/scala/io/constellationnetwork/node/shared/infrastructure/snapshot/CurrencySnapshotAcceptanceManager.scala

```diff
@@ -30,6 +31,7 @@ trait CurrencySnapshotAcceptanceManager[F[_]] {
   def accept(
     blocksForAcceptance: List[Signed[Block]],
     messagesForAcceptance: List[Signed[CurrencyMessage]],
+    sharedArtifactsForAcceptance: Option[SortedSet[SharedArtifact]],
```

Comment:
I've just spot it -> it should be without `Option` here too I think. take a look at `messagesForAcceptance`. Messages were added later, changing the schema. same for artifacts. take a look if it's possible, otherwise the current impl is lgtm :+1:

Reply:
@marcinwadon It's working as expected when changing from
`Option[SortedSet[SharedArtifact]]`
to
`SortedSet[SharedArtifact]`